core/backend/src/BriefcaseManager.ts

```diff
@@ -119,6 +119,8 @@ export class BriefcaseManager {
     if (this._initialized)
       return;
     this.setupCacheDir(cacheRootDir);
+    // disable this rule here since even though the method accesses "this", it is a static method accessing a static field of this class
+    // eslint-disable-next-line @typescript-eslint/unbound-method
     IModelHost.onBeforeShutdown.addOnce(this.finalize, this);
```

Comment:
```suggestion
    IModelHost.onBeforeShutdown.addOnce(() => this.finalize());
```
> it is a static method accessing a static field

`static` is irrelevant. `static` methods and field are just as vulnerable to unbound method errors as instance methods and fields.
In this case you are actually relying on the fact that `finalize` is currently `private`, so a subclass cannot override it - but that could change, or the code inside `finalize` could change.